docs/FAQ.md

```diff
@@ -0,0 +1,10 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+## FAQ
+
+#### Q. Which browsers is Accessibility Insights for Web available for?
```

Comment:
If we start this line with **Q.** it seems like it would make sense to start the answer line with **A.** - though probably I'd prefer not having either Q. or A. but just having the questions be headers.

Reply:
Agreed and made the changes. Github has this feature for "Siggested changes" where you can make suggestions for a change and the person who raised the PR can accept the suggestion in the PR itself. It does create a commit and push it to the branch for each change however. I just used it and it seems ideal for one, small number of lines change.